docs/spelling_wordlist

```diff
@@ -0,0 +1,26 @@
+addon
+Admin
+admin
+Aldryn
+app
+apphook
+apphooks
+Apps
+Backend
+Blog
+CMS
+Configuratons
+Divio
+django
+excecute
+Frontend
+frontend
+i
+placeholder
+placeholders
+plugins
+py
+settings
+setup
+toolbar
+weblog
```

Comment:
It's not an archaic term. Some people say _weblog_ and some people say _blog_, which is after all nothing more than a contraction of _weblog_.

https://www.djangoproject.com/weblog/ just for a start.